src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyGroupSizes.cs

```diff
@@ -13,9 +13,13 @@ public static IEnumerable<object[]> CurrencyGroupSizes_TestData()
         {
             yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 3 } };
             yield return new object[] { new CultureInfo("en-US").NumberFormat, new int[] { 3 } };
-            
-            if (!PlatformDetection.IsWindows7)
+
+            // TODO: when dotnet/corefx#2103 is addressed, we should also check fr-FR
+            if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610
```

Comment:
I do not know about Ubuntu, Fedora. I stay this code as it was before. I just add Windows 8 because it does not support culture "ur-IN" on my computer Windows Server 2012. Windows Server 2012 is server version of Windows 8.